data/org.freedesktop.impl.portal.ScreenCast.xml

```diff
@@ -239,9 +239,9 @@
 
         A bitmask of available source types. Currently defined types are:
 
-        - ``1``: MONITOR
-        - ``2``: WINDOW
-        - ``4``: VIRTUAL
+        - ``1``: MONITOR: Monitors (non-virtual)
+        - ``2``: WINDOW: Application windows
+        - ``4``: VIRTUAL: Virtual monitors
```

Comment:
Perhaps we should be slightly more verbose:

```
        - ``1``: MONITOR: Share existing monitors
        - ``2``: WINDOW: Share application windows
        - ``4``: VIRTUAL: Extend with new virtual monitor
```

or something

Because `1` can still mean "virtual" monitor, just that something else created it.

Reply:
I applied the suggestion.